components/Python/Scikit-learn/dataframe_to_file/component.json

```diff
@@ -0,0 +1,40 @@
+{
+  "engineType": "Generic",
+  "language": "Python",
+  "userStandalone": false,
+  "name": "dataframe_to_file",
+  "label": "Sink DataFrame to File",
+  "program": "dataframe_to_file.py",
```

Comment:
add description that the component writes DF to file, and passes filename to the next component